src/main/java/emissary/config/ConfigUtil.java

```diff
@@ -431,7 +429,7 @@ public static InputStream getConfigStream(final String name) throws IOException
         File f = new File(sname);
```

Comment:
Consider starting with a `Path` here and then using the static `Files` api method `isReadable` with checks for both existence and readability.

Reply:
.. there are a bunch of other cases where we could work with `Path` instead of `File` objects and the `Files` api methods in this class. I won't annotate each one. I wonder if we could drop all imports of `java.io.File`.